src/channels/components/AssignmentList/styles.ts

```diff
@@ -20,18 +20,30 @@ export const useExpanderStyles = makeStyles(
   { name: "Expander" },
 );
 
-export const useHeaderStyles = makeStyles(
-  theme => ({
+export const useHeaderContainerStyles = makeStyles(
```

Comment:
Y tho
It's like creating three different `.css` files (each hook creates separate `<styles/>` tag) so it's slower and seems to me like over the top separation of concerns. There are cases where I'd advice to split stylesheets, eg. reusing the same styles in different components or during component splitting, but having each class in different hooks is not one these cases.

Reply:
I did that because when you pass a class key that is not supported by material UI it raises errors in the console. However, you are right here, and to avoid these errors I can just pass the only classes that are needed